src/Servers/Kestrel/Core/src/Internal/Infrastructure/TransportManager.cs

```diff
@@ -57,14 +60,81 @@ public async Task<EndPoint> BindAsync(EndPoint endPoint, MultiplexedConnectionDe
         // The transport will check if the feature is missing.
```

Comment:
Does "This should always be set in production" need to be clarified?

Reply:
I've updated the comment to be clearer.